eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -193,37 +197,34 @@ public void exec() throws IOException {
     /**
      * Transpile.
      *
-     * @param file The path to the .xmir file
-     * @return Path to transpiled .xmir file
+     * @param input The .xmir file
+     * @param target The path to transpiled .xmir file
+     * @return Count of saved java files
      * @throws IOException If any issues with I/O
      */
-    public Path transpile(final Path file) throws IOException {
-        final XML input = new XMLDocument(file);
+    public int transpile(final XML input, final Path target) throws IOException {
         final String name = input.xpath("/program/@name").get(0);
         final Place place = new Place(name);
-        final Path target = place.make(
-            this.targetDir.toPath().resolve(TranspileMojo.DIR),
-            TranspileMojo.EXT
+        Train<Shift> trn = TranspileMojo.TRAIN;
```

Comment:
@Karina5005 I wonder, why this variable `trn` is not `final`?

Reply:
It can be final. Fixed